### PR TITLE
Add USWDS

### LINKED ISSUE #392: Set up USWDS
**Background**
Get the design system into the app early to be the foundation for rapid prototyping.  

**We'll know we're done when...**
- [x] The elements in existing prototype conform to USWDS style
- [x] The list of grants looks like a table or standard USWDS list

**Tasks**
- [x] Add style and JS into Angular build pipeline

### PR DESCRIPTION
**Pull request by beechnut**
Adds USWDS to build, and redesigned sample data to be a sortable table.
Closes #392 

Screen Shot 2021-08-16 at 4 44 30 PM

### CODE REVIEW

File: api/src/index.ts [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 
 /**
  * @todo Set using environment variables for security hardening.
  */
-app.use(function(req, res, next) {
-  res.setHeader('Access-Control-Allow-Origin',  '*')
-  res.setHeader('Access-Control-Allow-Methods', '*')
-  res.setHeader('Access-Control-Allow-Headers', '*')
+app.use(function (req, res, next) {
+  res.setHeader("Access-Control-Allow-Origin", "*");
```
Comment (neilmb) [inline api/src/index.ts]: I think that these formatting changes appeared in another PR, possibly a merge conflict coming?

File: frontend/src/app/app.component.ts [hunk 2]
```diff
@@ -6,5 +6,5 @@ import { Component } from '@angular/core';
   styleUrls: ['./app.component.css'],
 })
 export class AppComponent {
-  title = 'NRM G&A';
+  title = 'Grants & Agreements';
```
Comment (neilmb) [inline frontend/src/app/app.component.ts:9]: I think that there is a test for this string in particular. Will this fail? (If we are actually running those tests first?)

**Review by neilmb — DISMISSED** [review]
**suggestion (non-blocking):** I think that it would have been nice to see a screenshot of the new USWDS-ified table in the PR description.

File: frontend/src/app/app.component.ts (hunk 2 above)
Comment (beechnut) [reply frontend/src/app/app.component.ts:9]: Good catch!

File: api/src/index.ts (hunk 1 above)
Comment (beechnut) [reply api/src/index.ts]: Good memory. Seems like we'll find out soon!

**Comment by beechnut** [comment]
> **suggestion (non-blocking):** I think that it would have been nice to see a screenshot of the new USWDS-ified table in the PR description.

@neilmb Done!

**Review by neilmb — DISMISSED** [review]

**Review by neilmb — APPROVED** [review]

**Pull request merged**